rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.69"
+channel = "1.70"
```

Comment:
I wonder if this change is really needed? If it is, the ci jobs and dockefile also need to update to this rust version.

Reply:
I did get a stack overflow in cases::test_openbook_v2::test_openbook_compute with 1.69, so seems needed.